pyresample/geometry.py

```diff
@@ -2743,9 +2727,10 @@ def geocentric_resolution(self, ellps='WGS84', radius=None):
 def _get_area_boundary(area_to_cover: AreaDefinition) -> Boundary:
     try:
         if area_to_cover.is_geostationary:
-            return Boundary(*get_geostationary_bounding_box_in_lonlats(area_to_cover))
-        boundary_shape = max(max(*area_to_cover.shape) // 100 + 1, 3)
-        return area_to_cover.boundary(frequency=boundary_shape, force_clockwise=True)
+            vertices_per_side = None
+        else:
+            vertices_per_side = max(max(*area_to_cover.shape) // 100 + 1, 3)
+        return area_to_cover.boundary(vertices_per_side=vertices_per_side, force_clockwise=True)
```

Comment:
Here before when geostationary area, we were not forcing to be `clockwise` !

Reply:
Doesn't the geostationary boundary computation always return clockwise anyway?

Reply:
For GOES and Himawari yes. For SEVIRI I am not sure ... as the data are flipped right?
BTW now we enforce to be clockwise ... just to be sure ;)

Reply:
ok, fine by me :) but seviri is rotated 180, so still clockwise